.github/workflows/kfp-kubernetes-test-python38.yaml

```diff
@@ -0,0 +1,28 @@
+name: kfp kubernetes tests with Python 3.8
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - '.github/workflows/kfp-kubernetes-test-python38.yaml'
+      - 'sdk/python/**'
+      - 'api/v2alpha1/**'
+      - 'kubernetes_platform/**'
+
+jobs:
+  kfp-kubernetes-test-python38:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
```

Comment:
Looks like the KFP deploy step might be required here, and is missing. Could you confirm? If it's required, use this as reference: https://github.com/kubeflow/pipelines/pull/10926/commits/1737ef511431ebdc795e3b9315f03cecde44f61b#diff-e842340381d5ec7517ff5c2522b081e6b081cad5701ce9f06af7d088428974eeR29

Reply:
It is not required.